scripts/test_tools.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2025 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This script tests multiple tools with multiple markets."""
+
+import json
+import os
+from datetime import datetime
+from timeit import default_timer as timer
+
+from dotenv import load_dotenv  # type: ignore
```

Comment:
I wonder what we are ignoring here. Is it because of library stubs or something?